src/config.rs

```diff
@@ -48,12 +61,40 @@ impl Default for Config {
             traverse_parent_directories: false,
             exclude: Vec::new(),
             user_agent: default_user_agent(),
+            http_headers: Vec::new(),
             warning_policy: WarningPolicy::Warn,
             cache_timeout: Config::DEFAULT_CACHE_TIMEOUT.as_secs(),
         }
     }
 }
 
+impl TryFrom<String> for HttpHeader {
```

Comment:
Should we be implementing `std::are::FromStr` instead of `TryFrom`? 

I'm not 100% sure what the convention is, or even if there is one, seeing as `FromStr` is just a specialised version of `TryFrom`.

Reply:
This is not about "what trait fits better", this is about how `#[serde(try_from)]` works. At least, this is how I figured it out, feel free to correct me.